apps/condo/domains/common/components/Loader.tsx

```diff
@@ -0,0 +1,47 @@
+/** @jsx jsx */
+import React from 'react'
+import { Spin, SpinProps } from 'antd'
+import { css, jsx } from '@emotion/core'
+import { colors } from '../constants/style'
+
+const DEFAULT_DELAY = 200 // milliseconds
+
+interface ILoaderProps extends SpinProps {
+    fill?: boolean
+    color?: string
+}
+
+export const Loader: React.FC<ILoaderProps> = (props) => {
+
+    const filledContainerStyles = css`
```

Comment:
please don't use css call inside components

Reply:
Moved the filledContainer outside of the component and greatly simplified the code